src/pages/søknad/steg/bo-og-opphold-i-norge/Bo-og-opphold-i-norge.tsx

```diff
@@ -330,6 +368,59 @@ const BoOgOppholdINorge = (props: { forrigeUrl: string; nesteUrl: string }) => {
                                 feil={formik.errors.ektefellePartnerSamboer}
                             />
                         )}
+
+                        <RadioGruppe
+                            legend={intl.formatMessage({ id: 'input.adresse.tittel' })}
+                            feil={formik.errors.borPåAdresse}
+                        >
+                            <p className={styles.undertittelForAdresse}>
+                                {intl.formatMessage({ id: 'input.adresse.undertittel' })}
+                            </p>
+                            {adresser.map((a) => (
+                                <div className={styles.adresse} key={a.radioValue.adresselinje}>
+                                    <RadioPanel
+                                        label={a.label}
+                                        name="ingenAdresseGrunn"
+                                        onChange={() =>
+                                            formik.setValues({
+                                                ...formik.values,
+                                                borPåAdresse: a.radioValue,
+                                                ingenAdresseGrunn: null,
+                                            })
```

Comment:
Vi bør venne oss til å bruke callback-versjonen av `setValues`.

```suggestion
                                            formik.setValues((v) => ({
                                                ...v,
                                                borPåAdresse: a.radioValue,
                                                ingenAdresseGrunn: null,
                                            }))
```

Reply:
Gjelder også flere plasser nedover her.